ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -110,6 +110,25 @@ Item {
                 font.weight: Font.Bold
                 font.pixelSize: 20
             }
+
+
+            Component {
+              id: mailserversList
+
+              RadioButton {
+                text: name
+                checked: index == 0 ? true: false
```

Comment:
Looking at your code, what you need is a function call on the button click that tells Nim that you changed the mailserver and let the QML handle the selected index using a variable. Then you can check if the selected button is equal to the index.